packages/backend/src/models/AnalyticsModel.ts

```diff
@@ -136,6 +138,10 @@
             chartWeeklyAverageQueriesSql(userUuids, projectUuid),
         );
 
+        const dashboardViews = await this.database.raw(
+            dashboardViewsSql(projectUuid),
```

Comment:
This is how we are doing requests for analytics, if we need to refactor this I'd refactor all the things, instead of making changes only to these new methods